#!/bin/sh /usr/share/dpatch/dpatch-run

## 01_debian.dpatch by Darren Salt
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Package-specific bits

@DPATCH@
diff -uNr vdr-plugin-xine-0.9.0.orig/Make.config vdr-plugin-xine-0.9.0/Make.config
--- vdr-plugin-xine-0.9.0.orig/Make.config	1970-01-01 01:00:00.000000000 +0100
+++ vdr-plugin-xine-0.9.0/Make.config	2009-01-05 21:37:47.000000000 +0100
@@ -0,0 +1,5 @@
+INCLUDES+=-I/usr/include/freetype2
+VDRDIR=/usr/include/vdr
+LIBDIR=$(DESTDIR)/usr/lib/vdr/plugins
+VIDEODIR=/var/lib/video
+DATA_DIR=$(DESTDIR)/usr/share/vdr-plugin-xine
diff -uNr vdr-plugin-xine-0.9.0.orig/Makefile vdr-plugin-xine-0.9.0/Makefile
--- vdr-plugin-xine-0.9.0.orig/Makefile	2009-01-05 21:37:34.000000000 +0100
+++ vdr-plugin-xine-0.9.0/Makefile	2009-01-05 21:39:06.000000000 +0100
@@ -31,7 +31,7 @@
 
 ### Allow user defined options to overwrite defaults:
 
--include $(VDRDIR)/Make.config
+-include Make.config
 
 INCLUDES += `pkg-config --cflags libxine` 
 
@@ -48,9 +48,10 @@
 #VDR_XINE_SET_VIDEO_WINDOW = 1
 
 # where are these utilities for image grabbing? (default: anywhere on your PATH)
-#VDR_XINE_Y4MSCALER = /usr/bin/y4mscaler
-#VDR_XINE_Y4MTOPPM  = /usr/bin/y4mtoppm
-#VDR_XINE_PNMTOJPEG = /usr/bin/pnmtojpeg
+VDR_XINE_Y4MSCALER = /usr/bin/y4mscaler
+VDR_XINE_Y4MTOPPM  = /usr/bin/y4mtoppm
+VDR_XINE_PNMTOJPEG = /usr/bin/pnmtojpeg
+
 
 ### The version number of VDR's plugin API (taken from VDR's "config.h"):
 
@@ -112,7 +113,7 @@
 ### Internationalization (I18N):
 
 PODIR     = po
-LOCALEDIR = $(VDRDIR)/locale
+LOCALEDIR = $(DESTDIR)$(VDRDIR)/locale
 I18Npo    = $(wildcard $(PODIR)/*.po)
 I18Nmo    = $(addsuffix .mo, $(foreach file, $(I18Npo), $(basename $(file))))
 I18Ndirs  = $(notdir $(foreach file, $(I18Npo), $(basename $(file))))
@@ -140,6 +141,16 @@
 	$(CXX) $(CXXFLAGS) -shared $(OBJS) -o $@
 	@cp --remove-destination $@ $(LIBDIR)/$@.$(APIVERSION)
 
+install: libvdr-$(PLUGIN).so
+	@install -D -m755 $< $(LIBDIR)/$<.$(APIVERSION)
+	@install -d "$(DATA_DIR)"
+	@install -m644 data/* "$(DATA_DIR)/"
+
+uninstall:
+	@rm $(LIBDIR)/$<.$(APIVERSION)
+	@rm $(addprefix "$(DATA_DIR)/", $(patsubst data/%,%,$(wildcard data/*)))
+	@rmdir --ignore-fail-on-non-empty "$(DATA_DIR)"
+
 xineplayer: xineplayer.o Makefile
 	$(CXX) $(CXXFLAGS) xineplayer.o -o $@
 
